src/DDTrace/Tracer.php

```diff
@@ -383,6 +383,17 @@ public function getTracesAsArray()
         }
 
         $internalSpans = dd_trace_serialize_closed_spans();
+
+        // Setting global tags on internal spans, if any
```

Comment:
Should we remove this (and the test for it) since it's part of a separate PR? :)

Reply:
actually I pulled in that branch, so once we merge that one this will disappear.